routers/api/v1/user/follower.go

```diff
@@ -86,13 +90,17 @@ func ListFollowers(ctx *context.APIContext) {
 }
 
 func listUserFollowing(ctx *context.APIContext, u *user_model.User) {
-	users, err := user_model.GetUserFollowing(u, utils.GetListOptions(ctx))
+	users, err := user_model.GetUserFollowing(ctx, user_model.GetUserFollowOptions{
+		Actor:         ctx.Doer,
+		RequestedUser: u,
+		ListOptions:   utils.GetListOptions(ctx),
+	})
 	if err != nil {
 		ctx.Error(http.StatusInternalServerError, "GetUserFollowing", err)
 		return
 	}
 
-	ctx.SetTotalCountHeader(int64(u.NumFollowing))
+	ctx.SetTotalCountHeader(int64(len(users)))
```

Comment:
No idea, if it's incorrect it was already broken behavior. Not the matter of this PR to fix tbh.

Reply:
nvm ... a sound in my mind is: removing it since it's incorrect and doesn't seem to be useful .... 😂

Reply:
Huh? TotalCount should always list the overall amount, not just the pagesize. Look at other references of the function

Reply:
Yup, so that's my question: is `len(users))` the overall amount and correct?

* If yes: all correct
* If no:
    * if it's useful, should it be fixed?
    * if it's not useful, remove it, or leave it as incorrect?

Reply:
> No idea, if it's incorrect it was already broken behavior. Not the matter of this PR to fix tbh.

Please just fill a new issue for this or open a PR if you think it's incorrect. Again not the scope of the PR. Just changing as according to the new behavior.